src/index.js

```diff
@@ -72,6 +71,35 @@ class Hue {
         const color = getRandomColor()
         this.setColors(ids, color)
     }
+
+    updateLight = async (id, state) => {
```

Comment:
I thought about adding this, but it would require you to keep track of the light's state somewhere. However, if you did, it would save a lot of api calls. Because you could check if the next state will be the state it already is, saving a lot of power imo

Reply:
This is a great idea for the future!